queries.sql

```diff
@@ -1,3 +1,71 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+/* Attached are images of the queries */
```

Comment:
Great job @MarcoDiaz2000 🤝

- [ ] You have done a great job implementing the changes. However, you have added the screenshots to the `queries.sql` file instead of adding them to your pull request. As stated by the previous reviewer, could you add the screenshots for your query results to your pull request description?